src/ucp/api/device/ucp_host.h

```diff
@@ -156,6 +156,17 @@ ucp_device_mem_list_create(ucp_ep_h ep,
                            ucp_device_mem_list_handle_h *handle);
 
 
+/**
+ * @ingroup UCP_DEVICE
+ * @brief Getter for mem list length.
+ *
+ * This function returns number of elements in the descriptors mem list handle.
```

Comment:
```suggestion
 * @brief Return the number of elements in the descriptors mem list handle.
```
It seems that `brief` is enough.